public/faq.html

```diff
@@ -238,9 +237,49 @@ <h4 id="faq8" class="card-category card-title text-left " style="color: #66615b"
                          The software to be installed can be found here: <a href="https://www.medicaldataworks.nl/security.html#requirements">IT requirements</a>. The free version of Docker is sufficient.
                          <br>
                     </p>
-
-                    <br>
-            </div>
+                    <h4 id="node-network-requirements" class="card-category card-title text-left " style="color: #66615b">
+                         What are some typical network requirements for a Node (station)?
+                    </h4>
+                    <p class="card-category card-title text-left " style="color: #66615b">
+                        <b>Network requirements for a node</b>
```

Comment:
This <b>Network requirements for a node</b> 'heading' kind of line looks a bit strange to me in the context of the rest of the layout, looks to me like the line could just be removed. Obviously a bit nitpicky so feel free to ignore.